system-tests/test/record_spec.js

```diff
@@ -2689,7 +2689,7 @@ describe('capture-protocol api errors', () => {
 
         expect(artifactReport?.protocol).to.exist()
         expect(artifactReport?.protocol?.error).to.equal(
-          'Failed to upload Test Replay: 500 Internal Server Error (http://localhost:1234/capture-protocol/upload/?x-amz-credential=XXXXXXXX&x-amz-signature=XXXXXXXXXXXXX)',
+          'Failed to upload Test Replay after 1 attempts. Errors: 500 Internal Server Error (http://localhost:1234/capture-protocol/upload/?x-amz-credential=XXXXXXXX&x-amz-signature=XXXXXXXXXXXXX)',
```

Comment:
This should be singular, `attempt`:

```suggestion
          'Failed to upload Test Replay after 1 attempt. Errors: 500 Internal Server Error (http://localhost:1234/capture-protocol/upload/?x-amz-credential=XXXXXXXX&x-amz-signature=XXXXXXXXXXXXX)',
```

Reply:
fixed in https://github.com/cypress-io/cypress/pull/29745/commits/366f1d94a0a9e8c3b00105350410a6686bd1da0c